XenAdmin/Plugins/Features/TabPageFeature.cs

```diff
@@ -320,7 +318,8 @@ private void Browser_DocumentCompleted(object sender, WebBrowserDocumentComplete
             if (!XenCenterOnly && lastBrowserState != null)
             {
                 log.DebugFormat("url for '{0}' set to '{1}'", Helpers.GetName(lastBrowserState.Obj), e.Url);
-                lastBrowserState.Urls = new List<Uri> { e.Url };
+                if (ResettingPage == false)

```

Comment:
Code style: in C# we normally write: `if (myVariable)` instead of `if (myVariable == true)` and `if (!myVariable)` instead of `if (myVariable ==false)`. (There are another couple of instances of this below).

Reply:
Thanks a lot for sharing the tip.